core/arch/x86/decode_table.c

```diff
@@ -5326,6 +5327,426 @@ const instr_info_t prefix_extensions[][12] = {
     {EVEX_W_EXT,   0x660f7a10, "(evex_W ext 50)", xx, xx, xx, xx, xx, mrm|evex, x, 50},
     {EVEX_W_EXT,   0xf20f7a10, "(evex_W ext 60)", xx, xx, xx, xx, xx, mrm|evex, x, 60},
   },
+  { /* prefix extension 160 */
+    {INVALID,        0x389810, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID,      0xf3389810, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID,      0x66389810, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID,      0xf2389810, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID,        0x389810, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID,      0xf3389810, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {VEX_W_EXT,    0x66389818, "(vex_W ext 0)", xx, xx, xx, xx, xx, mrm|vex, x, 0},
```

Comment:
Are there AVX-512 opcodes with these other prefixes?  If not, I would think a vex-vs-evex table and reqp would be better than having 10 bad entries in each of these.

Reply:
When I started adding these I honestly wasn't sure. Now after doing a few more spot checks it looks like there _may_ be no other three-byte same-opcode versions with different prefix (can't rule out there there isn't one). I honestly would like to avoid re-factoring everything above, adding this was several hours of tedious work. I could add a comment mentioning that it may be able to compress this into e_vex_extensions.

Reply:
You know what, I'll do it, I think it's not that much more work actually. I think there are no other ones, I spot checked a few opcodes.

Reply:
Done.